examples/temperature-measurement-app/esp32/main/temperature-measurement.zap

```diff
@@ -0,0 +1,1171 @@
+{
+  "writeTime": "Tue Nov 17 2020 19:19:54 GMT+0100 (Central European Standard Time)",
+  "featureLevel": 4,
+  "creator": "zap",
+  "keyValuePairs": [
+    {
+      "key": "commandDiscovery",
+      "value": "1"
+    },
+    {
+      "key": "defaultResponsePolicy",
+      "value": "always"
+    },
+    {
+      "key": "manufacturerCodes",
+      "value": "0x1002"
+    }
+  ],
+  "package": [
+    {
+      "pathRelativity": "relativeToZap",
+      "path": "../../../../../third_party/zap/repo/zcl-builtin/silabs/zcl.json",
```

Comment:
when will these become parameters to compilation?

Reply:
I don't think those paths really matters when you are just generating the file fwiw. I have used files with hardcoded path to @jepenven-silabs hard drive and it was working.

The issue was while I was trying to edit the file and then save it. It was complaining about the path in this case.